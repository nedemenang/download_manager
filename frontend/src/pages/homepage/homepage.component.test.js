import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import HomePage from "./homepage.component";
import CardList from "../../components/card-list/card-list.component";
import InputBox from "../../components/input-box/input-box.component";

configure({ adapter: new Adapter() });
const enzymeWrapper = shallow(<HomePage />);
describe("<HomePage />", () => {
  test("should matches the stored snapshot", () => {
    expect(enzymeWrapper.text()).toMatchSnapshot();
  });

  test("should render Homepage", () => {
    expect(enzymeWrapper).toBeDefined();
  });

  test("should render cardlist", () => {
    expect(enzymeWrapper.find(CardList)).toBeDefined();
  });

  test("should render inputbox", () => {
    expect(enzymeWrapper.find(InputBox)).toBeDefined();
  });
});
