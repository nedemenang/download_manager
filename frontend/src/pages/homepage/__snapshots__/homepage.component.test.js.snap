// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> should matches the stored snapshot 1`] = `"Stears Movie Downloader"`;
